miio/philips_moonlight.py

```diff
@@ -241,7 +241,9 @@ def set_brightness_and_rgb(self, brightness: int, rgb: Tuple[int, int, int]):
             if color < 0 or color > 255:
                 raise PhilipsMoonlightException("Invalid color: %s" % color)
 
-        return self.send("set_brirgb", [brightness, rgb_to_int(rgb)])
+        params = list(rgb)
+        params.append(brightness)
```

Comment:
I think `[brightness, *rgb]` would also work here, but this is fine to me, so feel free to merge as you wish.

Reply:
Yep. Work like a charm!